app/routes.py

```diff
@@ -1 +1,196 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, abort, make_response, request
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from datetime import datetime
+import os, requests
+
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+
+def validate_item_by_id(model, id):
+    try:
+        id = int(id)
+    except:
+        abort(make_response({"message":f"{model.__name__} {id} invalid"}, 400))
+
+    item = model.query.get(id)
+
+    return item if item else abort(make_response({"message":f"{model.__name__} {id} not found"}, 404))
```

Comment:
Powerful `validate_item_by_id` helper function! 💯 

I also appreciate the one-line `if/else` statement and the use of Python's built-in `.__name__` variable here.